packages/c/css-scope-inline.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "css-scope-inline",
+  "filename": "script.min.js",
+  "description": "Lightweight CSS scoping for inline styles without a build step",
+  "keywords": [
+    "css",
+    "scope",
+    "inline",
+    "styling"
+  ],
+  "homepage": "https://github.com/gnat/css-scope-inline",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/gnat/css-scope-inline.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/answerdotai/css-scope-inline.git",
```

Comment:
As with #1830, I am not comfortable using a fork of the source repository as the auto-update target here, as there is no guarantee the source code isn't being tampered with. Please work with the original author to have it tagged or published to NPM instead.